index.html

```diff
@@ -0,0 +1,117 @@
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <title>Horiseon | Your Marketing Experts</title>
+  </head>
+  <body>
+    <!--Header with nav bar-->
+    <header class="header">
+      <h1>Hori<span class="seo">seo</span>n</h1>
+      <!--Nav bar with page links-->
+      <nav>
+        <ul>
+          <li>
+            <a id="" href="#search-engine-optimization"
+              >Search Engine Optimization</a
+            >
+          </li>
+          <li>
+            <a id="" href="#online-reputation-management"
+              >Online Reputation Management</a
+            >
+          </li>
+          <li>
+            <a id="" href="#social-media-marketing">Social Media Marketing</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!--Background image-->
+    <div class="hero"></div>
+    <!--3 main page sections-->
+    <main>
+      <section class="services">
+        <img
+          src="./assets/images/search-engine-optimization.jpg"
+          class="float-left"
+          alt="notebook with drawings of SEO-related terms"
+        />
+        <h2>Search Engine Optimization</h2>
+        <p>
+          The dominance of mobile internet use means that users are searching
+          for the right business as they travel, shop, or sit on their couch at
+          home. Search Engine Optimization (SEO) allows you to increase your
+          visibility and find the right customers for your business.
+        </p>
+      </section>
+      <section class="services">
+        <img
+          src="./assets/images/online-reputation-management.jpg"
+          class="float-right"
+          alt="laptop displaying 'reputation' and various charts"
+        />
+        <h2>Online Reputation Management</h2>
+        <p>
+          The web is full of opinions, and some of these can be negative. Social
+          media allows anyone with an internet connection to say whatever they
+          want about your business. Online Reputation Management gives you the
+          control over what potential customers see when they search for your
+          business.
+        </p>
+      </section>
+      <section class="services">
+        <img
+          src="./assets/images/social-media-marketing.jpg"
+          class="float-left"
+          alt="birdseye view over table covered with social media icons and words"
+        />
+        <h2>Social Media Marketing</h2>
+        <p>
+          Social media continues to have a sizable influence on buying habits.
+          Social media marketing helps you determine which platforms are suited
+          to your brand, using analytics to find the right markets and increase
+          your lead generation.
+        </p>
+      </section>
+    </main>
+    <!--Benefits sidebar-->
+    <aside class="benefits">
+      <div class="benefit">
```

Comment:
These `div` tags could of been `section` tags, this would of made your code more semantic.